src/synonym_keyphrase_candidates.json

```diff
@@ -0,0 +1,777 @@
+{
+    "synonym_info": [{
```

Comment:
Only comment here is it might make more sense to get rid of the nesting, and just make it an array (since I know you're not a big fan of long json lines ;) )